### PR TITLE
Fixes name of accented math symbol, eliminates duplicate newStandardHspace assignment

### LINKED ISSUE #1274: Incorrect atomType name in unicode-symbols.lua
```
local atomType = {
  ordinary = 0,
  bigOperator = 1,
  binaryOperator = 2,
  relationalOperator = 3,
  openingSymbol = 4,
  closeSymbol = 5,
  punctuationSymbol = 6,
  inner = 7,
  overSymbol = 8,
  underSymbol = 9,
  accentSymbol = 10,
  radicalSymbol = 11,
  vcenter = 12
}

local mkAtomType = {
  mathopen = atomType.openingSymbol,
  mathclose = atomType.closeSymbol,
  mathord = atomType.ordinary,
  mathfence = atomType.ordinary,
  mathalpha = atomType.ordinary,
  mathbin = atomType.binaryOperator,
  mathaccent = atomType.accentedSymbol,
  mathaccentwide = atomType.accentSymbol,
  mathrel = atomType.relationalOperator,
  mathunder = atomType.underSymbol,
  mathover = atomType.overSymbol,
  mathop = atomType.ordinary,
}
```

"mathaccent = atomType.accentedSymbol," -> accentedSymbol doesn't exist; this should presumably be accentSymbol. But I'm not familiar enough with SILE to know if this would break something.

### PR DESCRIPTION
**Pull request by arichard4**
Closes #1274

### CODE REVIEW

File: packages/math/base-elements.lua [hunk 1]
```diff
@@ -1237,7 +1237,6 @@ elements.atomType = atomType
 elements.scriptType = scriptType
 elements.mathVariantToScriptType = mathVariantToScriptType
 elements.symbolDefaults = symbolDefaults
-elements.newStandardHspace = newStandardHspace
 elements.newSubscript = newSubscript
 elements.newUnderOver = newUnderOver
 elements.newStandardHspace = newStandardHspace
```
Comment (arichard4) [inline packages/math/base-elements.lua:1242]: The removed assignment gets overwritten here

**Review by simoncozens — APPROVED** [review]
LGTM, thank you!

**Comment by arichard4** [comment]
Fixed commit message to correct format, per commitlint

**Comment by OlivierNicole** [comment]
I'm merging this then. Thank you!

**Pull request merged**